package.json

```diff
@@ -8,8 +8,10 @@
     "preview": "vite preview",
     "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
     "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
+    "check:lint": "eslint --max-warnings=0 --ignore-path .eslintignore ./src",
+    "check:prettier": "prettier -c ./src/",
```

Comment:
actually think you need to remove ./src here - otherwise lint-staged won't work as expected